youtube_to_anki/anki.py

```diff
@@ -7,35 +7,44 @@
 from tempfile import TemporaryDirectory
 from typing import Dict, Iterable
 
+import cv2
 from genanki import Deck, Model, Note, Package
 from pydub import AudioSegment
 
+style = """
+.card {text-align:center;}
+.card {font-family: myfont; font-size: 30px}
+"""
```

Comment:
unrelated change - having the text a bit bigger and aligned in the center by default makes previewing the output of the program a bit easier!

I usually use this to quickly preview the files and check that they are ok, and then convert them to a different type.